src/Infrastructure.EntityFramework/Repositories/OrganizationRepository.cs

```diff
@@ -91,6 +91,29 @@ public async Task<ICollection<OrganizationAbility>> GetManyAbilitiesAsync()
         }
     }
 
+    public async Task<ICollection<Core.Entities.Organization>> SearchUnassignedToProviderAsync(string name, string ownerEmail, int skip, int take)
```

Comment:
Same as my comment on the sproc - can this be simplified and less nested? Compare to the `ProviderRepository.Search` EF method.

Reply:
I've simplified it, it does look a lot better.